Source/Code/Kepler/Relativity.Kepler.Template/Services/Services.csproj

```diff
@@ -48,17 +48,17 @@
     <Reference Include="Microsoft.VisualStudio.CoreUtility">
       <Private>False</Private>
     </Reference>
-    <Reference Include="Relativity.API, Version=10.3.170.1, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Relativity.Api.10.3.170.1\lib\net462\Relativity.API.dll</HintPath>
+    <Reference Include="Relativity.API, Version=10.3.191.8, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
why is this version 10.3?

Reply:
We want this to be version 10.3 because per Kepler documentation, kepler is only supported past 10.3.191.8. Please refer to this page - https://platform.relativity.com/RelativityOne/index.htm#Kepler_framework/Template_for_Kepler_services.htm